extensions/sql-database-projects/src/models/project.ts

```diff
@@ -277,7 +346,33 @@ export class ProjectEntry {
 	}
 }
 
+/**
+ * Represents a database reference entry in a project file
+ */
+class DatabaseReferenceProjectEntry extends ProjectEntry {
+	constructor(uri: Uri, public databaseLocation: DatabaseReferenceLocation, public name?: string) {
+		super(uri, '', EntryType.DatabaseReference);
+	}
+}
+
 export enum EntryType {
 	File,
-	Folder
+	Folder,
+	DatabaseReference
+}
+
+export enum DatabaseReferenceLocation {
+	sameDatabase,
+	differentDatabaseSameServer
+}
+
+export enum SqlPlatforms {
```

Comment:
is this name accurate? this seems like CompatLevels

Reply:
changed this to TargetPlatform to match SSDT